### PR TITLE
merging PajaritoConicSolver and PajaritoSolver

### LINKED ISSUE #20: merge PajaritoConicSolver and PajaritoSolver
I think we should merge ``PajaritoConicSolver`` and ``PajaritoSolver``. This way we have a single set of options to document and users don't need to think about which version they're using. We just need to implement:
```
MathProgBase.NonlinearModel(s::PajaritoSolver) = PajaritoNLPModel(...)
MathProgBase.ConicModel(s::PajaritoSolver) = PajaritoConicModel(...)
```
This is one of the reasons why I pushed through the restructuring of the MathProgBase interface in the fall.

### LINKED ISSUE #21: enable GLPK as lazy solver
https://github.com/mlubin/Pajarito.jl/blob/e97766b1df96a1e74e4ce0adaa3294cf5924167a/test/runtests.jl#L29

### PR DESCRIPTION
**Pull request by emreyamangil**
This PR merges the conic and nonlinear solver.

Closes #20 
Closes #21

### CODE REVIEW

File: src/conic.jl [hunk 1]
```diff
@@ -22,12 +22,13 @@
...
      convex atoms defined in Convex.jl library.
- Both ways PajaritoConicSolver must be provided as the
+ Both ways PajaritoSolver must be provided as the
  preferred solver with a corresponding mip_solver and 
- conic_solver specified.
+ cont_solver specified.
 =========================================================#
 
 using JuMP
+include("solver.jl")
```
Comment (mlubin) [inline src/conic.jl]: Why include solver.jl twice? Just add it to Pajarito.jl

Comment (emreyamangil) [inline src/conic.jl]: Done!!

File: src/solver.jl [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#===================================================
+ This file implements the default PajaritoSolver
+===================================================#
+
+export PajaritoSolver
+immutable PajaritoSolver <: MathProgBase.AbstractMathProgSolver
+    verbose
```
Comment (mlubin) [inline src/solver.jl]: Could you document these options?

Comment (emreyamangil) [inline src/solver.jl]: Done!! Added them to readme too!

**Comment by mlubin** [comment]
Ok, let's wait for travis to pass now.

**Comment by mlubin** [comment]
Looks like things are passing. Could you squash the commits into one before we merge?

**Comment by mlubin** [comment]
(``git rebase -i HEAD~4`` and then set the last 3 commits to ``fixup``)

**Comment by emreyamangil** [comment]
Did it work?! I think I again messed up the force push :( (I forgot to force it and instead did a merge)

**Comment by emreyamangil** [comment]
But on the bright side seems like commit 9f8ee7a has everything in it :)

**Comment by mlubin** [comment]
:+1:

**Pull request merged**